aws_lambda_events/src/dynamodb/attributes.rs

```diff
@@ -169,7 +29,7 @@ mod test {
 
         let attr: AttributeValue = serde_json::from_value(value.clone()).unwrap();
         match attr {
-            AttributeValue::String(ref s) => assert_eq!("value", s.as_str()),
+            AttributeValue::S(ref s) => assert_eq!("value", s.as_str()),
```

Comment:
In my humble opinion this is a regression in readability. Fully spelled out variant names with the appropriate `#[serde(rename = "…")]` attributes seemed better.